docker-compose.dev.yml

```diff
@@ -0,0 +1,82 @@
+version: "3.11"
+services:
+  postgres:
+    container_name: postgres-triplane
+    image: postgres:14.6
+    hostname: postgres
+    ports:
+      - "5432:5432"
+      - "5433:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: triplane
+      POSTGRES_DB: postgres-triplane
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    restart: on-failure
+    networks:
+      - my_network
```

Comment:
WE shouldnt use network in docker compose if we want to all containers use the same one, because docker during build makes default network for every container.